rosidl_default_runtime/package.xml

```diff
@@ -9,7 +9,6 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <build_export_depend>rosidl_generator_cpp</build_export_depend>
```

Comment:
Shouldn't this be replaced with the runtime package?

Reply:
As this is the defaullt should export both, right?

package.xml
```xml
  <build_export_depend>rosidl_runtime_c</build_export_depend>
  <build_export_depend>rosidl_runtime_cpp</build_export_depend>
```

CMakeLists.txt
```make
ament_export_dependencies(rosidl_runtime_c)
ament_export_dependencies(rosidl_runtime_cpp)
```

Reply:
`package.xml`: yes.

`CMakeLists.txt`: I would say no, since that happens dynamically in the CMake extra file using the ament index.